pkg/frontend/frontend.go

```diff
@@ -324,6 +324,8 @@ func (f *frontend) chiAuthenticatedRoutes(router chi.Router) {
 				// Top nodes metrics endpoint
 				r.Get("/top/nodes", f.getAdminTopNodes)
 
+				r.Get("/effectiveroutingtables", f.getAdminOpenshiftClusterEffectiveRouteTable)
```

Comment:
```suggestion
				r.Get("/effectiveroutetable", f.getAdminOpenshiftClusterEffectiveRouteTable)
```

Suggesting this change for alignment with the verbiage used elsewhere throughout the code.